docs/en/Contributing-Getting-Started.md

```diff
@@ -0,0 +1,261 @@
+---
+layout: global
+title: Contributor Getting Started
+nickname: Contributor Getting Started
+group: Resources
+---
+
+Welcome to the Alluxio community! We are excited for your contributions and engagement in the
+community! Here is a guide on how to get started with becoming a member in the Alluxio open source
+community.
+
+## Requirements
+
+The main requirement is a computer with Mac OSX or Linux installed. Alluxio does not have Windows support at the moment.
+
+## Software and Account Preparation
+
+There are several accounts and setup required before contributing to Alluxio.
+
+### Java 7 or 8
+
+Developing Alluxio requires Java 7 or 8. If you are unsure of which version of java you have, you
+can check with this command:
+
+```bash
```

Comment:
I considered that, but I think the danger is that if the code is not consistent with the context around the code, the documentation could be incorrect. This makes it difficult when the different languages are being updated at different times.